src-docs/src/views/inline_edit/inline_edit_example.js

```diff
@@ -67,13 +78,42 @@ export const InlineEditExample = {
       demo: <InlineEditTitle />,
       props: { EuiInlineEditTitle },
     },
+    {
+      title: 'Customizing read and edit modes',
+      text: (
+        <>
+          <p>
+            Customize the <EuiCode>readMode</EuiCode> empty button by passing{' '}
+            <strong>EuiInlineEdit</strong> the <EuiCode>readModeProps</EuiCode>{' '}
+            property. <EuiCode>readMode</EuiCode> accepts{' '}
+            <EuiCode>EuiButtonEmpty</EuiCode> properties with the exception of{' '}
+            <EuiCode>onClick</EuiCode>.
+          </p>
+
+          <p>
+            Customize the <EuiCode>editMode</EuiCode> state by passing{' '}
+            <strong>EuiInlineEdit</strong> the <EuiCode>editModeProps</EuiCode>{' '}
+            property. These properties are applied directly to the{' '}
```

Comment:
Same as above, slight copy tweak of succinctness
```suggestion
            Customize the <EuiCode>editMode</EuiCode> state by passing{' '}
            <EuiCode>editModeProps</EuiCode>{' '}
            property. These properties are applied directly to the{' '}
```